python-packages/order_utils/test/test_doctest.py

```diff
@@ -8,7 +8,13 @@
 
 
 def test_all_doctests():
-    """Gather zero_ex.* modules and doctest them."""
+    """Gather zero_ex.* modules and doctest them.
+
+    NOTE: we now use `importlib` instead of `imp` because `imp` is deprecated
```

Comment:
I appreciate this content, but I think its too noisy for a code comment, and instead should be in a comment on the PR or in the commit message.  And, I think you could just say "`imp` is deprecated" and leave it at that.

Reply:
Though, there never was any mention of "`imp`" in the code; care to clarify?

Reply:
Of course. So, `walk_packages` returns a generator that crawls the module path. Eventually, it will yield a `ModuleInfo` tuple. The first argument (what is defined as `loader` in your for loop) is of type `importlib.abc.FileLoader`. This loader inherits from [Loader](https://docs.python.org/3/library/importlib.html#importlib.abc.Loader.load_module) whose documentation specifies that `load_module` is outdated (the reason this happens is that it uses `imp` module internally).

Documentation [here](https://docs.python.org/3/library/importlib.html#importlib.abc.Loader.load_module) does suggest to use `exec_module` instead but I failed in getting that to work (I tried `exec_module` and `create_module()` with no success). What did work is `importlib.import_module` so I ended up using that. Happy to try alternatives if you prefer!

Reply:
Whatever works!  I just want to make sure that there's some logical continuity between the code we had in place and what we're saying is deprecated.

Reply:
sounds great! I will remove the comment from here, and I will ensure it's documented in the PR. In that comment, instead of mentioning `imp` which is internal, I'll simply specify that `load_module` is deprecated so that it's clearer.